docs/guide/tutorial-console.md

```diff
@@ -165,7 +166,7 @@ class HelloController extends Controller
 Now, you can use the following syntax to run the command:
 
 ```
-./yii hello -m=hello
```

Comment:
Yes, but all other examples has the following syntax (without `./`):
```
yii ...
```
http://www.yiiframework.com/doc-2.0/guide-tutorial-console.html
I thought it would be nice to bring everything to same view.